src/orders/components/OrderAddressFields/OrderAddressFields.tsx

```diff
@@ -62,26 +65,11 @@ const OrderAddressFields: React.FC<OrderAddressFieldsProps> = ({
   };
 
   return (
-    <>
-      {isDraft && (
-        <OrderCustomerAddressesEditDialog
-          open={action === "edit-customer-addresses"}
-          variant={AddressEditDialogVariant.CHANGE_CUSTOMER}
-          {...addressFieldCommonProps}
-        />
-      )}
-
-      <OrderCustomerAddressesEditDialog
-        open={action === "edit-shipping-address"}
-        variant={AddressEditDialogVariant.CHANGE_SHIPPING_ADDRESS}
-        {...addressFieldCommonProps}
-      />
-      <OrderCustomerAddressesEditDialog
-        open={action === "edit-billing-address"}
-        variant={AddressEditDialogVariant.CHANGE_BILLING_ADDRESS}
-        {...addressFieldCommonProps}
-      />
-    </>
+    <OrderCustomerAddressesEditDialog
+      open={!!getVariant(action)}
```

Comment:
This isn't easily readable for me, I'd add explicit `const isAddressEditDialogVariantSelected = !!getVariant(action)` or something similar